dvc/repo/experiments/init.py

```diff
@@ -239,18 +253,16 @@ def init(
         except MissingParamsFile:
             pass
 
```

Comment:
Nitpicky: let's just specify those live_paths here:
```
    models = context.get("models")
    live = context.get("live")
    live_metrics = os.path.join(live, "metrics") if live else None
    live_plots = os.path.join(live, "scalars") if live else None
```

And, then you can specify them in `repo.stage.create` as following:
```python
 repo.stage.create(
        ...,
        metrics_no_cache=compact([context["metrics"], live_metrics]),
        plots_no_cache=compact([context["plots"], live_plots]),
)
```
 
 Makes it simpler, and avoids from saving it to the list.